contentcuration/contentcuration/management/commands/setup.py

```diff
@@ -122,6 +123,7 @@ def handle(self, *args, **options):
         channel1.main_tree.children.first().copy_to(channel4.main_tree)
 
         # Get validation to be reflected in nodes properly
+        ContentNode.objects.all().update(complete=True)
```

Comment:
This is the line that I expected to see - the other changes have implications for test behaviour, so I'd rather they were left as is.

Reply:
The remaining diff is just cleanup of the code I added some time ago and that is now obsolete. `generate_tree` is called before we start marking nodes as complete/incomplete.

I am planning to come with a way of how to test complete/incomplete in the next PR I am working on so that we can test self-healing of incorrectly marked nodes. I'd prefer not to leave it here as is because it's confusing - it looks like that you can set those values to chosen ones when you actually can't as it will be overridden by newly introduced scripts.

Reply:
> I am planning to come with a way of how to test complete/incomplete in the next PR

It can be done this way at this point:
https://github.com/MisRob/studio/commit/1fddf265ac457de6283ea6624eeb24e3827cc573

Reply:
I made some of these edits, as some of these utils are used during our tests, so I wanted to make sure the tests were accurately reflecting appropriate values for complete. e.g. https://github.com/learningequality/studio/commit/ac072ea4350a0d0367b821160634ef3d982aa7e4

Reply:
Ah okay, so the issue is rather db utils parameters, not devsetup data itself. I haven't noticed that we are using utils from somewhere else - will revert that part.